evalml/model_understanding/graphs.py

```diff
@@ -947,9 +1034,29 @@ def graph_partial_dependence(
                     x=part_dep["feature_values"],
                     y=part_dep["partial_dependence"],
                     name="Partial Dependence",
-                    line=dict(width=3),
+                    line=dict(width=3, color="rgb(99,110,250)"),
                 )
-            return _go.Figure(layout=layout, data=[trace])
+            fig.add_trace(trace)
+    return fig
+
+
+def _add_ice_plot(_go, fig, ice_data):
+    x = ice_data["feature_values"]
+    if "class_label" in ice_data.columns:
+        ice_data.drop(columns=["class_label"], inplace=True)
+    ice_data.drop(columns=["feature_values"], inplace=True)
+    for i, sample in enumerate(ice_data):
+        fig.add_trace(
+            _go.Scatter(
+                x=x,
+                y=ice_data[sample],
+                line=dict(width=0.5, color="gray"),
```

Comment:
There's some leeway to adjust this but from tweaking it a bit this is the best color and thickness combo for the ICE plot lines.